csrc/ops/composite.cpp

```diff
@@ -584,6 +584,15 @@ SdpfaBwdResult sdpfa_bwd(
   NVF_CHECK(
       !scale || scale->isScalar(), "Expected scale to be a scalar double.");
 
+  // Set default values for dropout_p (0.0), is_causal(false)
+  if (dropout_p == nullptr) {
+    dropout_p = IrBuilder::create<Val>(0.0, DataType::Double);
+  }
+
+  if (is_causal == nullptr) {
+    is_causal = IrBuilder::create<Val>(false, DataType::Bool);
```

Comment:
nitpick we have zeroVal and falseVal for this.
https://github.com/NVIDIA/Fuser/blob/862dfd18a03b48617c4789b7507455eb996fb5d3/csrc/ir/container.h#L134